tests/Issues/IssueDowngradeThrowableType/config/configured_rule.php

```diff
@@ -3,10 +3,13 @@
 declare(strict_types=1);
 
 use Rector\Config\RectorConfig;
+use Rector\Core\ValueObject\PhpVersion;
 use Rector\DowngradePhp70\Rector\FunctionLike\DowngradeThrowableTypeDeclarationRector;
 use Rector\DowngradePhp74\Rector\ArrowFunction\ArrowFunctionToAnonymousFunctionRector;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->rule(ArrowFunctionToAnonymousFunctionRector::class);
     $rectorConfig->rule(DowngradeThrowableTypeDeclarationRector::class);
+
+    $rectorConfig->phpVersion(PhpVersion::PHP_70);
```

Comment:
It seems it should be PHP 5.6 instead, ref https://github.com/rectorphp/rector-downgrade-php/blob/c5e04ca5255452f1f9fdab28b8a6bcb7676f24a0/config/set/downgrade-php70.php#L27